scripts/gulpfile.factory.js

```diff
@@ -271,18 +244,24 @@ ${fs
   .join("")}*/\n\n`
         )
       )
-      .pipe(dest(paths.compiled));
+      .pipe(dest(paths.temp));
   });
 
-  task("copy", () => {
-    return src(["*.js", `!${elementName}.js`], {
```

Comment:
I don't know why `!${elementName}.js` is here, lol.  I guess it's there because `merge` is already "copying" the files into the destination dir.

Reply:
Right the `elementName*.js` is being handled by the `merge` task but this allows copying of the other dependencies such as `icon-set.js` to be copied over to temp for inclusion during rollup.